changes/1630.changes

```diff
@@ -0,0 +1 @@
+Remove warning alert from the Contracts PD.
```

Comment:
@RabiaSajjad u can do `1630.removal` which will mark this PR in our dev change logs as a code removal.